Readme.md

```diff
@@ -0,0 +1,135 @@
+
+# First-Capstone
+
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Capstone-Project: Summer BootCamp 2023 ](#-capstone-project-summer-bootcamp-2023-)
+  - [🛠 Built With ](#-built-with-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Capstone-Project: Summer Music Festival <a name="about-project"></a>
+
+**[Capstone-Project: Summer Music Festival]** is about series of conscert prodution during Summer. the program will feature all different kind of music and artist .
+
+[Loom Video Presentation](https://www.loom.com/share/fec156053a8d4a5488170b52f51be1be)
+
```

Comment:
[REQUIRED]You did an amazing job on the video how ever you missed out key points to talk about in your video presentation. It would be nice if you talked about a feature you added that you are most proud of. Kindly base your video presentation on the following requirements.

The student recorded a video presenting the features of the project.
The maximum length of the video is 5 minutes.
The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
The video contains a demo of the project, showing the pages in different screen sizes.
The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

I strongly recommend you kindly redo the video  based on the above requirements for it is part of project requirements.